### PR TITLE
Meals by time

### PR DESCRIPTION
**Pull request by yuvalbardugo**

### CODE REVIEW

File: api/explore/explore.js [hunk 1]
```diff
@@ -76,25 +103,33 @@ async function GetBoxes(size, dateString, request, amount) {
...
   const {
     size = 10,
-    dateString = '',
+    Country = '',
+    timeString = '',
     amount = 30,
   } = request.body;
 
-  if (!search.isString(dateString)) {
+  if (!search.isString(Country)) {
```
Comment (hagaishapira) [inline api/explore/explore.js:111]: what about timeString then

File: api/explore/explore.js [hunk 2]
```diff
@@ -76,25 +103,33 @@ async function GetBoxes(size, dateString, request, amount) {
 router.post('/', asyncHandler(async (request, response, next) => {
   const {
     size = 10,
-    dateString = '',
+    Country = '',
```
Comment (hagaishapira) [inline api/explore/explore.js:106]: @LeshemNoa Are you okay with this interface?

File: api/explore/explore.js [hunk 3]
```diff
@@ -30,13 +31,38 @@ async function innerSearch(searchString, amount, request) {
...
-async function GetBoxes(size, dateString, request, amount) {
+function mealByTime(timeString) {
+  let nextMeal = '';
+  let greet = '';
+  const format = 'HH:mm:ss';
+  const time = moment(timeString, format);
+
+  if (time.isBetween(moment('05:00:00', format), moment('09:59:59', format), null, '[]')) {
+    nextMeal = 'Breakfast';
+    greet = 'Good Morning, try breakfast recipes:';
```
Comment (hagaishapira) [inline api/explore/explore.js:42]: Where is this displayed, not sure we should say Good Morning if it's in the middle of the page

Comment (yuvalbardugo) [reply api/explore/explore.js:42]: If it won't be at the top of the page, we will be able choose not to have the greet.
I think it should be at the top though...

**Comment by hagaishapira** [comment]
What's "Watercress"

**Pull request merged**